data-tool/flows/tombstone/tombstone_utils.py

```diff
@@ -785,7 +785,7 @@ def build_filing_json_meta_data(raw_filing_type: str, filing_type: str, filing_s
 
     if filing_type == 'annualReport':
         meta_data['annualReport'] = {
-            'annualReportFilingYear': int(effective_date[:4]),
+            'annualReportFilingYear': int(data['f_period_end_dt_str'][:4]),
```

Comment:
I'll think it would be better to update effective date to use `period_end_dt` for ARs and it's a special case.

The following screenshot is an AR filed in a different year of the AR year in LEAR.
![image](https://github.com/user-attachments/assets/3c03ba34-47b9-4591-b716-2712eacd7ce5)

Reply:
I don't think that's required. Even in Lear we have different date in effective_date and annualReportDate

Reply:
This is how we sync to colin. https://github.com/bcgov/lear/blob/main/colin-api/src/colin_api/models/filing.py#L613

Reply:
I try to trace the code logic back to FE,
1. https://github.com/bcgov/business-filings-ui/blob/main/src/views/AnnualReport.vue#L1033
2. https://github.com/bcgov/business-filings-ui/blob/main/src/views/AnnualReport.vue#L545
3. https://github.com/bcgov/business-filings-ui/blob/main/src/stores/businessStore.ts#L214